### PR TITLE
Adding smoove.cwl to run Lumpy

### PR DESCRIPTION
**Pull request by apaul7**

### CODE REVIEW

File: definitions/tools/smoove.cwl [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        inputBinding:
+            prefix: --exclude
+            position: 4
+        doc: "Bed file used to exclude regions"
+
+outputs:
+    output_vcf:
+        type: File
+        outputBinding:
+            glob: "$(inputs.cohort_name)-smoove.genotyped.vcf.gz"
```
Comment (tmooney) [inline definitions/tools/smoove.cwl]: Does smoove produce a tabix index with this that we'd want as a secondary file?

Comment (apaul7) [reply definitions/tools/smoove.cwl]: yes. updating and testing it now

**Comment by apaul7** [comment]
Would it be better to change the reference input to a string instead of a file so it can work with the current workflows?

**Comment by tmooney** [comment]
We'll need to change it to a string if we add this step to one of our existing workflows, yes.

**Pull request merged**